airflow/providers/facebook/ads/hooks/ads.py

```diff
@@ -118,11 +125,32 @@ def bulk_facebook_report(
         :param sleep_time: Time to sleep when async call is happening
         :type sleep_time: int
 
-        :return: Facebook Ads API response, converted to Facebook Ads Row objects
-        :rtype: List[AdsInsights]
+        :return: Facebook Ads API response,
+            converted to Facebook Ads Row objects regarding given Account ID type
```

Comment:
Probably want to describe a bit more what account ID type triggers what kinds of results. It may be a good idea to provide a couple of examples above.